airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -77,7 +77,7 @@ def get_date_params(stream_slice: Mapping[str, Any], cursor_field: str, end_date
         if start_date > pendulum.now():
             return start_date.to_date_string(), start_date.add(days=1).to_date_string()
 
-        end_date = min(end_date, pendulum.parse(stream_slice.get(cursor_field)).add(months=1))
+        end_date = max(end_date, pendulum.parse(stream_slice.get(cursor_field)).add(months=1))
```

Comment:
If we accept your changes, then we will get a lot of unnecessary duplicates.
The code now works in such a way that we read data in one month.
For example, start_date is 2021-03-18, then we will have several requests:

| start_date   |      end_date      | 
|----------|:-------------:|
| 2021-03-18 |  2021-04-18 |
| 2021-04-19 |  2021-05-18 |
| 2021-05-19 |  2021-06-18 |
| 2021-06-19 |  2021-07-18 |
| 2021-07-19 |  2021-07-18 | 

**This is where the code breaks down**

If we well use your changes, then the following situation will be:
| start_date   |      end_date      | 
|----------|:-------------:|
| 2021-03-18 |  2021-07-18 |
| 2021-04-19 |  2021-07-18 |
| 2021-05-19 |  2021-07-18 |
| 2021-06-19 |  2021-07-18 |
| 2021-07-19 |  2021-08-18 | 

The problem exists, but change `min` to `max` is not the correct solution.